crates/tauri-runtime/src/webview.rs

```diff
@@ -400,6 +405,19 @@ impl WebviewAttributes {
     self
   }
 
+  /// Set the path from which to load extensions from. Extensions stored in this path should be unpacked.
+  ///
+  /// ## Platform-specific:
+  ///
+  /// - **Windows**: Browser extensions must first be enabled. See [`browser_extensions_enabled`](Self::browser_extensions_enabled)
+  /// - **Linux**: Browser extensions do not need to be enabled.
+  /// - **MacOS / iOS / Android** - Unsupported.
+  #[must_use]
+  pub fn set_extension_path(mut self, path: impl AsRef<Path>) -> Self {
```

Comment:
I feel like this should be called `extensions_path` to convey that it may load more than one extensions
```suggestion
  pub fn extensions_path(mut self, path: impl AsRef<Path>) -> Self {
```